gef.py

```diff
@@ -6203,22 +6203,23 @@ def pprint_bin(arena_addr: str, index: int, _type: str = "") -> int:
             warn("Invalid backward and forward bin pointers(fw==bk==NULL)")
             return -1
 
-        if gef.libc.version >= (2, 34):
-            if index >= 1:
-                previous_bin_address = arena.bin_at(index-1)
-                if previous_bin_address == fd:
-                    return 0
+        if _type == "tcache":
```

Comment:
👍

Reply:
Since we want specific cases such as GlibcTcacheChunk & GlibcFastChunk, it is required to modify the private attribute to be public.